### PR TITLE
fix rebalance in facade

### PR DESCRIPTION
**Pull request by julianmrodri**
* Add try catch, this is required for `nextRecollateralizationAuction` to work, otherwise the call to `_rebalance` reverts when fully collateralization was reached.

### CODE REVIEW

**Review by tbrent — COMMENTED** [review]
Good catch. One other place we also need to touch if we want to maintain the property of not reverting.

File: contracts/facade/FacadeAct.sol [hunk 1]
```diff
@@ -253,7 +253,8 @@ contract FacadeAct is IFacadeAct, Multicall {
         bytes1 majorVersion = bytes(bm.version())[0];
 
         if (majorVersion == MAJOR_VERSION_3) {
-            bm.rebalance(TradeKind.DUTCH_AUCTION);
+            // solhint-disable-next-line no-empty-blocks
+            try bm.rebalance(TradeKind.DUTCH_AUCTION) {} catch {}
         } else if (majorVersion == MAJOR_VERSION_2 || majorVersion == MAJOR_VERSION_1) {
             IERC20[] memory emptyERC20s = new IERC20[](0);
             address(bm).functionCall(
```
[the remark this answers is not in the document]
Comment (julianmrodri) [reply contracts/facade/FacadeAct.sol:260]: Ok will double check this. I think we had this in prior versions

**Comment by julianmrodri** [comment]
confirmed we had in fact an anomaly in behavior when there were no auctions to be run, `nextRecollateralizationAuctions`   reverted for 3.0 but not for 2.1. Now its fixed, It never reverts and returns false.

**Comment by tbrent** [comment]
LGTM

**Pull request merged**